### ISSUE TITLE
Question about transfer learning

### ISSUE THREAD

**Issue by Izzysh7** [microsoft/RegionCLIP#29]
impressive work! I feel confused about transfer learning.  What does "when human annotations of objects are available" means?  lable or bounding box?
when we make evaluation for "zero-shot inference", we set "MODEL.CLIP.CROP_REGION_TYPE GT ". why we use GT before transfer learning?  In which situation will we set  "MODEL.CLIP.CROP_REGION_TYPE GT "?
and when we Train detectors on my own, at train_transfer_learning.sh, there is nothing about "human annoutations"?

thank you for your kindly help

**Comment by YiwuZhong** [comment]
@Izzysh7 After pre-training, our model can already supports open-set object detection. If detection labels (human annotated boxes and their categorical labels) are available, our model can be further fine-tuned.

For GT boxes, please take a look at the Setup of Section 4.2 in our paper. GT box setting aims at evaluating the recognition performance by eliminating the localization error. It is an evaluation setting and independent from transfer learning.

For transfer learning, the "human annotation" is already specified in the config file (eg, ./configs/COCO-InstanceSegmentation/CLIP_fast_rcnn_R_50_C4_ovd.yaml).

**Comment by mactavish91** [comment]
> @Izzysh7 After pre-training, our model already supports open-set object detection. If detection labels (human annotated boxes and their categorical labels) are available, our model can be further fine-tuned.
> 
> For GT boxes, please take a look at the Setup of Section 4.2 in our paper. GT box setting aims at evaluating the recognition performance by eliminating the localization error. It is an evaluation setting and independent from transfer learning.
> 
> For transfer learning, the "human annotation" is already specified in the config file (eg, ./configs/COCO-InstanceSegmentation/CLIP_fast_rcnn_R_50_C4_ovd.yaml).

If detection labels (human annotated boxes and their categorical labels) are available, how does the detector optimize the classification results? Because I observed that the parameters of the full connection layer used for classification are frozen.

**Comment by YiwuZhong** [comment]
@mactavish91 The classification layer is fixed while the previous layers get updated. You can find more details in Section 3.3 of paper.
